# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/moz.build
# Commit: a114b9b8e6ea
# Full Hash: a114b9b8e6eaa23a445a31efc436f4e367f7874f
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-01-10 21:49:10
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Convert IDBCursor and BackgroundCursorChild to templates to increase type safety and reduce state. r=dom-workers-and-storage-reviewers,ytausky
#   
#   This also simplifies delegating calls that are dependent on the cursor type.
#   
#   Also reduce dependency on IDBCursor.h by moving enums and type traits to IDBCursorType.h
# ==============================================================================

diff -r cc3cfee4a014 -r a114b9b8e6ea dom/indexedDB/moz.build
--- a/dom/indexedDB/moz.build	Fri Jan 10 11:07:59 2020 +0000
+++ b/dom/indexedDB/moz.build	Fri Jan 10 14:21:18 2020 +0000
@@ -26,6 +26,7 @@
 EXPORTS.mozilla.dom += [
     'FlippedOnce.h',
     'IDBCursor.h',
+    'IDBCursorType.h',
     'IDBDatabase.h',
     'IDBEvents.h',
     'IDBFactory.h',
@@ -39,6 +40,7 @@
     'IDBTransaction.h',
     'IndexedDatabase.h',
     'IndexedDatabaseManager.h',
+    'InitializedOnce.h',
 ]
 
 EXPORTS.mozilla.dom.indexedDB += [
@@ -56,6 +58,7 @@
     'FileInfo.cpp',
     'FileSnapshot.cpp',
     'IDBCursor.cpp',
+    'IDBCursorType.cpp',
     'IDBDatabase.cpp',
     'IDBEvents.cpp',
     'IDBFactory.cpp',
